server/util/grpc_server/grpc_server.go

```diff
@@ -82,5 +83,9 @@ func CommonGRPCServerOptions(env environment.Env) []grpc.ServerOption {
 		grpc.StreamInterceptor(grpc_prometheus.StreamServerInterceptor),
 		grpc.UnaryInterceptor(grpc_prometheus.UnaryServerInterceptor),
 		grpc.MaxRecvMsgSize(env.GetConfigurator().GetGRPCMaxRecvMsgSizeBytes()),
+		grpc.KeepaliveEnforcementPolicy(keepalive.EnforcementPolicy{
```

Comment:
can you add a comment here about why this is set (that includes the error / ENHANCE_YOUR_CALM strings?)

Reply:
Done